lib/livebook_web/live/session_live/cell_component.ex

```diff
@@ -253,7 +253,8 @@ defmodule LivebookWeb.SessionLive.CellComponent do
       <span class="font-mono"
         id={"cell-timer-#{@cell_view.id}-evaluation-#{@cell_view.number_of_evaluations}"}
         phx-hook="Timer"
-        phx-update="ignore">
+        phx-update="ignore"
+        data-start={DateTime.to_iso8601(@cell_view.evaluation_start)}>
```

Comment:
We already have evaluation-time-ms. Why do we need both start and the elapsed time?

Reply:
Elapsed time is measured in evaluator, so it is more precise. The start time is captured in Livebook as soon as we start the evaluation.

Reply:
Sorry, I am trying to understand why we need both in the front end. If we have evaluation-time-ms, we know the total time evaluating, right? Is it because we should other information that is based on data-start?

Reply:
Ah, it goes like this:

1. We start evaluation and set `start` to the current timestamp.
2. We ask the evaluator to evaluate.
3. We use `start` to show the ticking timer, with `start` as the point to count from.
4. We receive response from evaluator with the total `evaluation_tims_ms`
5. We show the green evaluated status with `evaluation_tims_ms` in the tooltip.

Reply:
Perfect.